src/pages/membership.tsx

```diff
@@ -0,0 +1,131 @@
+    const { data, classes } = props
+    const { boardBackground } = data
+    if (!boardBackground) throw new Error("Board background does not exist.")
+
+    return (
+        <>
+            <SEO title="Membership Cards" />
+            <ParallaxBackground image={boardBackground}>
+                <Text variant="h3" color="white" align="center">
+                    TASA Membership Cards
+                </Text>
+                <Text align="center">Valid through the 2022-2023 school year</Text>
+            </ParallaxBackground>
+
+            <PageContent>
+                <Section title="What is a TASA Membership Card?">
+                    <Text align="center">
+                        A TASA membership card brings fun, happiness, 
+                        and premium benefits right to your door. These 
+                        benefits include 10% off Cuppa Tea, Class520, and Aroy 
+                        Thai (more to come!!). In addition, this card 
+                        will grant you free event tickets and raffle 
+                        tickets at our major events (Tour of Taiwan and 
+                        Night Market)! For all of these perks, you only 
+                        have to pay a small price of $15 or bundle with 
+                        some friends by buying a group package!
+                    </Text>
+                </Section>
+                
+                <Section title="Current Benefits Include">
+                    <ul>
+                        <li>
+                            <Text>10% off at <a href="https://www.cuppatea.us/">
+                                Cuppa Tea
+                            </a>
+                            </Text>
+                            
+                        </li>
+                        <li>
+                            <Text>10% off at <a href="https://www.aroythaicollegepark.com/">
+                                Aroy Thai
+                            </a>
+                            </Text>
+                        </li>
+                        <li>
+                            <Text>10% off at <a href="https://www.class520cp.com/">
+                                Class520
+                            </a>
+                            </Text>
+                        </li>
+                        <li>
+                            <Text>Free Event Tickets to Tour of Taiwan and Night Market</Text>
+                        </li>
+                        <li>
+                            <Text>Free Raffle Tickets at Tour of Taiwan and Night Market</Text>
+                        </li>
+                    </ul>
+                </Section>
+
+                <Section title="Pricing">
+                    <Text>$15 for 1 card</Text>
+                    <Text>$27 for 2 cards</Text>
+                    <Text>$48 for 4 cards</Text>
+                    <Text>$100 for 10 cards</Text>
+                </Section>
+
+                <Section title="Form">
```

Comment:
perhaps change this to "Order Your Membership Card Today!" or something like that